src/ostree/ot-admin-builtin-pin.c

```diff
@@ -28,9 +28,74 @@
 
 static gboolean opt_unpin;
 
+typedef enum
+{
+  OSTREE_DEPLOYMENT_INDEX = 0,
+  OSTREE_DEPLOYMENT_BOOTED = 1,
+  OSTREE_DEPLOYMENT_PENDING = 2,
+  OSTREE_DEPLOYMENT_ROLLBACK = 3
+} OstreeDeploymentType;
+
 static GOptionEntry options[]
     = { { "unpin", 'u', 0, G_OPTION_ARG_NONE, &opt_unpin, "Unset pin", NULL }, { NULL } };
 
+static gint64
+get_deployment_index (const OstreeDeployment *target_deployment, const GPtrArray *deployments)
+{
+  for (gint i = 0; i < deployments->len; ++i)
+    if (deployments->pdata[i] == target_deployment)
```

Comment:
BTW our style generally prefers braces `{` in these types of cases, really braceless is only for a simple `if`, but this is irrelevant per above

Reply:
Add both to for and if I assume?

Reply:
The sort of vague unspecified rule I follow is that if you have any conditional/looping instructions that nest, then use braces at least on the outer one. 

So
```
for (...) {
  if (something)
    return i;
}
```
is OK too.  The rationale behind this is the same as that for [-Wmisleading-indentation](https://developers.redhat.com/blog/2016/02/26/gcc-6-wmisleading-indentation-vs-goto-fail?source=sso).